emails/models.py

```diff
@@ -375,27 +382,46 @@ def update_abuse_metric(self, address_created=False, replied=False):
             abuse_metric.num_address_created_per_day += 1
         if replied:
             abuse_metric.num_replies_per_day += 1
+        if email_forwarded:
+            abuse_metric.num_email_forwarded_per_day += 1
+        if forwarded_email_size > 0:
+            abuse_metric.forwarded_email_size_per_day += forwarded_email_size
         abuse_metric.last_recorded = datetime.now(timezone.utc)
         abuse_metric.save()
 
         # check user should be flagged for abuse
         hit_max_create = False
         hit_max_replies = False
+        hit_max_forwarded = False
```

Comment:
Nit: We're missing `hit_max_forwarded_email_size`, but I don't think we need any of the initializers

Reply:
Completed in: [81c5f02](https://github.com/mozilla/fx-private-relay/pull/2409/commits/81c5f0206cea99a1ed43e3eb70058473ba19f526)